src/scripts/compile.ts

```diff
@@ -77,12 +80,25 @@ export const registerHandlers = (types: Handler[]): { [type: string]: HandlerFun
 	}, {});
 };
 
+export const getFrontmatter = (content: string): { [key: string]: string } => {
+	const pipeline = unified()
+		.use(remarkParse, { commonmark: true })
+		.use(frontmatter, 'yaml')
+		.use(parseFrontmatter);
+
+	const nodes = pipeline.parse(content);
+	const result = pipeline.runSync(nodes);
+	const node = result.children.find((node: WNode) => node.type === 'yaml');
```

Comment:
I would be okay with casting this to `any` instead of `WNode` as neither of the 2 children will be `WNode`s. The content is always a `div` `VNode`.